bricks/next-builder/src/preview-container/PreviewContainer.tsx

```diff
@@ -90,6 +98,7 @@ export interface PreviewContainerRef {
   toggleTheme(): void;
   back(): void;
   forward(): void;
+  previewDataValue(name: string, option: PreviewDataOption): void;
```

Comment:
```suggestion
  inspectDataValue(name: string, option: PreviewDataOption): void;
```

其他的如果对外暴露的 api 名字都改下吧